code/game/machinery/portable_turret.dm

```diff
@@ -1041,6 +1041,9 @@ GLOBAL_LIST_EMPTY(turret_icons)
 	ailock = TRUE
 	var/area/syndicate_depot/core/depotarea
 
+/obj/machinery/porta_turret/syndicate/CanPass(atom/A)
+    return !isliving(A)
```

Comment:
could you return `TRUE` here if the turret is broken?

`stat & BROKEN` should be enough for that